ansible/roles/software/ckan/inventory/templates/etc/supervisor/conf.d/gunicorn-datapusher.conf

```diff
@@ -0,0 +1,10 @@
+[program:gunicorn-datapusher]
+command={{ datapusher_virtual_env }}/bin/gunicorn -b 127.0.0.1:6000 wsgi:app
```

Comment:
Similar to inventory, I think we want to use an async worker type. The default is probably fine for now though.